demo/util/util.mjs

```diff
@@ -0,0 +1,14 @@
+/**
+ * Replaces |swapOut| with |swapIn| in the DOM. Reparent all children
+ * and copy all attributes.
+*/
+export function swap(swapOut, swapIn) {
```

Comment:
swapElement or swapElementKeepingChildren would be better.

Reply:
done